modules/local/summarize_results/main.nf

```diff
@@ -0,0 +1,50 @@
+process SUMMARIZE_RESULTS {
+    label 'process_single'
+    tag "${meta.id}"
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mhcgnomes:1.8.6--pyh7cba7a3_0' :
+        'biocontainers/mhcgnomes:1.8.6--pyh7cba7a3_0' }"
+
+    input:
+    tuple val(meta), path(csv)
+
+    output:
+    tuple val(meta), path("*.tsv") , emit: tsv
+    tuple val(meta), path("*.json"), emit: json
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def wide_format_output = params.wide_format_output ? "--wide_format_output" : ""
```

Comment:
I know this is a local module, but the previous implementation passing it using the `modules.config` is more nf-core friendly, see [here](https://nf-co.re/docs/guidelines/components/modules#use-of-parameters-within-modules)